src/sql/workbench/browser/modelComponents/card.component.ts

```diff
@@ -2,7 +2,7 @@
  *  Copyright (c) Microsoft Corporation. All rights reserved.
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-import 'vs/css!./media/card';
+import 'vs/css!./media/legacycard';
```

Comment:
we are sure this wont brake any new card right?

Reply:
yes, I am only updating the card component, the radio card group remains the same